### DISCUSSION TITLE
Rack utilization API

### DISCUSSION THREAD

**Discussion by romanmendelproject** [netbox-community/netbox#9458] [Q&A]
Hi All!
How can I get rack utilization percentage via API?

**Comment by empusas** [comment]
I have the same requirement, but I would suggest to have the number of free height units instead of a percentage value. That is much more practical information, because otherwise you need to calculate the free units form the overall unit and the percentage value. 
I had a look at the core and the function seems to be there already. It is just a matter of adding it to the serializer.

**Reply by Emusp** [reply]
I agree, it would be really useful to get the number of free units for data center capacity planning.

**Reply by Razor648** [reply]
any updates please ? I also need the space utilization value for my capacity planning.

**Reply by empusas** [reply]
We have created a little plugin that extends the API with the missing fields for various endpoint as we could not wait and the discussion seems to be stuck.
